src/xsl/openrosa2html5form.xsl

```diff
@@ -403,38 +409,18 @@ XSLT Stylesheet that transforms OpenRosa style (X)Forms into valid HTMl5 forms
     </xsl:template>
 
     <xsl:template name="appearance">
-        <xsl:variable name="select-type">
-            <xsl:if test="local-name() = 'select' or local-name() = 'select1'">
-                <xsl:value-of select="'true'"/>
-            </xsl:if>
-        </xsl:variable>
-        <xsl:if test="@appearance">
-            <xsl:variable name="appearances" select="str:tokenize(@appearance)" />
-            <xsl:for-each select="exsl:node-set($appearances)">
-                <xsl:variable name="appearance">
-                    <xsl:value-of select="normalize-space(translate(., $upper-case, $lower-case))"/>
-                </xsl:variable>
-                <xsl:value-of select="concat('or-appearance-', $appearance, ' ')"/>
-                <!-- convert deprecated appearances, but leave the deprecated ones -->
-                <xsl:if test="$select-type = 'true'">
-                    <xsl:if test="$appearance = 'horizontal'">
-                        <xsl:value-of select="'or-appearance-columns '" />
-                    </xsl:if>
-                    <xsl:if test="$appearance = 'horizontal-compact'">
-                        <xsl:value-of select="'or-appearance-columns-pack '" />
-                    </xsl:if>
-                    <xsl:if test="$appearance = 'compact'">
-                        <xsl:value-of select="'or-appearance-columns-pack or-appearance-no-buttons '" />
-                    </xsl:if>
-                    <xsl:if test="starts-with($appearance, 'compact-')">
-                        <xsl:value-of select="concat('or-appearance-columns-', substring-after($appearance, '-'), ' or-appearance-no-buttons ')" />
-                    </xsl:if>
-                </xsl:if>
-            </xsl:for-each>
+        <xsl:variable name="rows" select="./@rows" />
```

Comment:
I haven't fully traced the `rows` logic. I'm counting on the test at https://github.com/enketo/enketo-transformer/blob/a231a5f9c3545949bdfe7214b0ffa6662cc96724/test/transformer.spec.ts#L665 to be enough to support the refactor.

In general, this appearance logic feels like an area of risk because I assume there aren't a ton of test forms with appearances (I could be wrong). I read through it enough to convince myself it was likely correct but if we agree it's risky I can spend more time with it.